### PR TITLE
Improved performance for exporting packedbubble series.

### PR DESCRIPTION
**Pull request by Izothep**
Improved performance for exporting packedbubble series.
___
With current packed bubbles series the process of exporting an image may take more than 6500ms. 

It was needed to add independent **seriesBbox** method, calculating bBox for bubble series without rendered elements. 

With new approach it was possible to reduce exporting time to 2100ms.

### CODE REVIEW

File: js/parts-more/PackedBubbleSeries.js [hunk 1]
```diff
@@ -705,6 +707,35 @@ seriesType(
...
+         * @private
+         */
+        seriesBox: function () {
+            var series = this,
+                data = series.data,
+                max = Math.max,
+                min = Math.min,
+                radius,
+                // bBox = [xMin, xMax, yMin, yMax]
+                bBox = [Infinity, -Infinity, Infinity, -Infinity];
```
Comment (pawelfus) [inline js/parts-more/PackedBubbleSeries.js]: How about we start from `[plotLeft, plotLeft + plotWidth, plotTop, plotTop + plotHeight]` ?

Comment (Izothep) [reply js/parts-more/PackedBubbleSeries.js]: good idea, much more clear approach

**Comment by pawelfus** [comment]
6.5s -> 2.1s ? With just a few lines of code? Great job!

File: js/parts-more/PackedBubbleSeries.js (hunk 1 above)
Comment (Izothep) [reply js/parts-more/PackedBubbleSeries.js]: Done with e25a2ab

**Pull request merged**